03_launch_mgmt_cluster.sh

```diff
@@ -18,64 +18,11 @@ sudo chown -R "${USER}:${USER}" "${IRONIC_DATA_DIR}"
 
 # Create certificates and related files for TLS
 if [ "${IRONIC_TLS_SETUP}" == "true" ]; then
-  export IRONIC_BASE_URL="https://${CLUSTER_URL_HOST}"
-
-  export IRONIC_CACERT_FILE="${IRONIC_CACERT_FILE:-"${WORKING_DIR}/certs/ironic-ca.pem"}"
-  export IRONIC_CAKEY_FILE="${IRONIC_CAKEY_FILE:-"${WORKING_DIR}/certs/ironic-ca.key"}"
-  export IRONIC_CERT_FILE="${IRONIC_CERT_FILE:-"${WORKING_DIR}/certs/ironic.crt"}"
-  export IRONIC_KEY_FILE="${IRONIC_KEY_FILE:-"${WORKING_DIR}/certs/ironic.key"}"
-
-  export IRONIC_INSPECTOR_CACERT_FILE="${IRONIC_INSPECTOR_CACERT_FILE:-"${WORKING_DIR}/certs/ironic-ca.pem"}"
-  export IRONIC_INSPECTOR_CAKEY_FILE="${IRONIC_INSPECTOR_CAKEY_FILE:-"${WORKING_DIR}/certs/ironic-ca.key"}"
-  export IRONIC_INSPECTOR_CERT_FILE="${IRONIC_INSPECTOR_CERT_FILE:-"${WORKING_DIR}/certs/ironic-inspector.crt"}"
-  export IRONIC_INSPECTOR_KEY_FILE="${IRONIC_INSPECTOR_KEY_FILE:-"${WORKING_DIR}/certs/ironic-inspector.key"}"
-
   pushd "${WORKING_DIR}"
   mkdir -p "${WORKING_DIR}/certs"
   pushd "${WORKING_DIR}/certs"
```

Comment:
that should be moved as part of the TLS setup

Reply:
Done